### DISCUSSION TITLE
Order of Books?

### DISCUSSION THREAD

**Discussion by NorPhi** [TriliumNext/Trilium#1790] [Q&A] [answered]
I might just overlooked it but is there an option to assign an order to the notes in a book?

**Comment by puffitos** [comment, marked as answer]
Right clicking on your Book / any note, and then _advanced_, provides a `Sort By...` option (or just click Alt+S). You may sort all children of said book by title or creation/modification date.

**Reply by NorPhi** [reply]
Good to know but unfortunately this doesn't allow later insertion of notes in between. I was hoping for something like assigning a priority to each note and then the option to sort by that.

**Reply by zadam** [reply]
Book note displays the notes in the same order as they are in the tree so you can just reorder them in the tree:

1. by drag & drop
2. clicking on the note in the tree and then using `CTRL-UP/DOWN/LEFT/RIGHT` to position the note

![image](https://user-images.githubusercontent.com/617641/112390827-2f1a5e00-8cf7-11eb-8f83-c4f76aada571.png)
